botorch/optim/initializers.py

```diff
@@ -468,6 +468,91 @@ def gen_batch_initial_conditions(
     return batch_initial_conditions
 
 
+def gen_optimal_input_initial_conditions(
```

Comment:
Could you add a docstring explaining the behavior of this function?

Reply:
Added!

Reply:
Thanks!